site/content/docs/cli/constructs-reference.md

```diff
@@ -77,6 +77,7 @@ derived from the abstract class `Check`.
 - `tags`: An array of tags to help you organize your Checks, i.e. `['product', 'api']`
 - `runtimeId`: The ID of which [runtime](https://www.checklyhq.com/docs/runtimes/specs/) to use for this Check.
 - `testOnly`: A boolean determining if the Check is available only when `test` runs and not included when `deploy` is executed.
+- `shouldFail`: A boolean that indicates whether the request should fail. This only works with API checks, it allows you to treat HTTP error codes (4xx and 5xx) as correct responses.
```

Comment:
This could go into the API Checks section below, then you can drop `This only works with API checks`. :)